data/lib/core/constants.lua

```diff
@@ -42,15 +42,25 @@ CYCLOPEDIA_SKILL_FISHING = 13
 
 CYCLOPEDIA_SKILL_AMOUNT = 8
 
-BLESS_TYPE_ADVENTURER = 1 -- not supported
-BLESS_TYPE_TWIST_OF_FATE = 2 -- not supported
-BLESS_TYPE_WISDOM_OF_SOLITUDE = 4
-BLESS_TYPE_SPARK_OF_PHOENIX = 8
-BLESS_TYPE_FIRE_OF_THE_SUNS = 16
-BLESS_TYPE_SPIRITUAL_SHIELD = 32
-BLESS_TYPE_THE_EMBRACE_OF_THE_WORLD = 64
-BLESS_TYPE_HEART_OF_MOUNTAIN = 128 -- not supported
-BLESS_TYPE_BLOOD_OF_MOUNTAIN = 256 -- not supported
+BLESSING_ID_ADVENTURERS_BLESSING = 0 -- not supported
+BLESSING_ID_TWIST_OF_FATE = 1 -- not supported
+BLESSING_ID_WISDOM_OF_SOLITUDE = 2
+BLESSING_ID_SPARK_OF_THE_PHOENIX = 3
+BLESSING_ID_FIRE_OF_THE_SUNS = 4
+BLESSING_ID_SPIRITUAL_SHIELDING = 5
+BLESSING_ID_EMBRACE_OF_THE_WORLD = 6
+BLESSING_ID_HEART_OF_THE_MOUNTAIN = 7 -- not supported
+BLESSING_ID_BLOOD_OF_THE_MOUNTAIN = 8 -- not supported
+
+BLESSING_FLAG_ADVENTURER = 1 -- not supported
+BLESSING_FLAG_TWIST_OF_FATE = 2 -- not supported
+BLESSING_FLAG_WISDOM_OF_SOLITUDE = 4
+BLESSING_FLAG_SPARK_OF_PHOENIX = 8
+BLESSING_FLAG_FIRE_OF_THE_SUNS = 16
+BLESSING_FLAG_SPIRITUAL_SHIELD = 32
+BLESSING_FLAG_THE_EMBRACE_OF_THE_WORLD = 64
+BLESSING_FLAG_HEART_OF_MOUNTAIN = 128 -- not supported
+BLESSING_FLAG_BLOOD_OF_MOUNTAIN = 256 -- not supported
```

Comment:
`FLAG` can always be obtained by calculating `1 << ID`, we can try to have only one of these